tests/smoke-test/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "type": "module",
+  "name": "@sap-ai-sdk/smoke-test",
+  "version": "0.0.0",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "private": "true",
+  "scripts": {
+    "start": "node dist/server.js",
+    "precompile": "pnpm create-deployment",
+    "compile": "tsc",
+    "create-deployment": "rm -rf deployment && cp -r ../../sample-code/src ./deployment",
+    "cf-login": "dotenv -- bash -c 'cf login -a \"$CF_API_URL\" -u \"$CF_USER\" -p \"$CF_PASSWORD\" -o \"$CF_ORG\" -s \"$CF_SPACE\"'",
+    "test": "NODE_OPTIONS=--experimental-vm-modules jest",
+    "lint": "prettier . --config ../../.prettierrc --ignore-path ../../.prettierignore -c",
+    "lint:fix": "prettier . --config ../../.prettierrc --ignore-path ../../.prettierignore -w --log-level error"
```

Comment:
I belive eslint is missing from lint:fix

Reply:
I guess we also missed that in the e2e-tests directory, unless it was intentional

Reply:
I think for e2e tests I just copied the lint command from somewhere else. Is this not the right command? 😄

Reply:
It's partially correct, the follow command chains with eslint, here is the command from one of our packages:
```
"eslint . --fix && prettier . --config ../../.prettierrc --ignore-path ../../.prettierignore -w --log-level error
```